src/arrays/multidimensional_arrays/hard/q6/MainTest.java

```diff
@@ -0,0 +1,102 @@
+package arrays.multidimensional_arrays.hard.q6;
+
+import global.BaseTest;
+import global.MethodTest;
+import global.tools.CustomAssertions;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest extends BaseTest {
+    // Parsons
+
+    public static final int [][] a = { {4, 5, 6}, {66, 54, 21}, {609, 998, 543}, {1111, 2342, 4583}  };
+    public static final int [][] b = { {4433, 2316, 9879}, {662, 154, 721}, {69, 98, 54},{1, 4, 3}  };
+
+    public Clause[] testSentence() {
+        if(a.length != b.length || a[0].length != b[0].length) {
+            return new Clause[]{
+                    new StringLiteral("Can't add because arrays of different sizes!")
+            };
+        }
+        else{
+            Clause[] c = new Clause[3 * a.length];
+            int[][] ans = answerFor(a, b);
+            for(int i = 0; i < a.length; i ++){
+                String st = "";
+                for(int j = 0; j < a[0].length; j ++){
+                    st += ans[i][j] + " ";
+                }
+                c[3 * i] = new StringLiteral(st.trim());
```

Comment:
Since this is Parsons, the students won't have the option of using .trim() and therefore the added " " at the end of a line will always be there